scripts/build-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
It seems that travis-sphinx is designed to push docs to the `gh_pages` branch rather than to a separate repository. Can easily change this later.